IL/MirrorSharpOptionsExtensions.cs

```diff
@@ -0,0 +1,23 @@
+using System;
+using Microsoft.CodeAnalysis;
+using MirrorSharp.IL;
+using MirrorSharp.IL.Internal;
+using MirrorSharp.Internal;
+
+// ReSharper disable once CheckNamespace
+namespace MirrorSharp
+{
```

Comment:
Brace does not follow end-of-line, same below. Not a blocker, I can adjust.

Reply:
Fixed!